postgres/views/gateway_availability.sql

```diff
@@ -0,0 +1,14 @@
+CREATE OR REPLACE VIEW app_monitoring_gateway_health AS (
+    SELECT
+        *,
+        CASE
+            WHEN due > 0 AND delivered = 0 THEN 'Gateway Unhealthy'
+            ELSE
+                'Gateway Healthy'
```

Comment:
There would be value in showing the actual number of days since the last delivered message vs scheduled time. This would be a great metric to get alerts on.

Reply:
I've added it with a caveat being we will only be able to know the `last_delivered` date if it falls within the function's time range i.e in this case 120 days in the past.